vsintegration/src/FSharp.VS.FSI/sessions.fs

```diff
@@ -130,37 +128,46 @@ let catchAll trigger x =
     try trigger x  
     with err -> System.Windows.Forms.MessageBox.Show(err.ToString()) |> ignore
 
-let fsiExeName () = 
-    if SessionsProperties.useAnyCpuVersion then "fsiAnyCpu.exe" else "fsi.exe"
-
-// Use the VS-extension-installed development path if available, relative to the location of this assembly
-let determineFsiRelativePath1 () =
-    let thisAssemblyDirectory = typeof<EventWrapper>.Assembly.Location |> Path.GetDirectoryName
-    Path.Combine(thisAssemblyDirectory,fsiExeName() )
-
-// This path is relative to the location of "FSharp.Compiler.Interactive.Settings.dll"
-let determineFsiRelativePath2 () =
-    let thisAssembly : System.Reflection.Assembly = typeof<FSharp.Compiler.Server.Shared.FSharpInteractiveServer>.Assembly
-    let thisAssemblyDirectory = thisAssembly.Location |> Path.GetDirectoryName
-    // Use the quick-development path if available    
-    Path.Combine(thisAssemblyDirectory,fsiExeName() )
-
 let determineFsiPath () =    
-    // Choose VS extension path, if it exists (for developers)
-    let fsiRelativePath1 = determineFsiRelativePath1()
-    if  File.Exists fsiRelativePath1 then fsiRelativePath1 else
-
-    // Choose relative path, if it exists (for developers), otherwise, the installed path.    
-    let fsiRelativePath2 = determineFsiRelativePath2()
-    if  File.Exists fsiRelativePath2 then fsiRelativePath2 else
-
-    // Try the registry key
-    let fsbin = match Internal.Utilities.FSharpEnvironment.BinFolderOfDefaultFSharpCompiler(None) with Some(s) -> s | None -> ""
-    let fsiRegistryPath = Path.Combine(fsbin, fsiExeName() )
-    if File.Exists(fsiRegistryPath) then fsiRegistryPath else
-
-    // Otherwise give up
-    raise (SessionError (VFSIstrings.SR.couldNotFindFsiExe fsiRegistryPath))
+    if SessionsProperties.fsiUseNetCore then 
+        let exe = @"c:\Program Files\dotnet\dotnet.exe"
```

Comment:
Combine the ProgramFiles Environment variable with dotnet\dotnet.exe
Set ProgramFiles
ProgramFiles=C:\Program Files

Reply:
Yup will do

Reply:
(Need to use this https://weblog.west-wind.com/posts/2019/Feb/05/Finding-the-ProgramFiles64-Folder-in-a-32-Bit-App as we're in a 32-bit app)